build.sh

```diff
@@ -242,7 +241,7 @@ if [ "$DO_UPDATE" == true ] || [ ! -f $HTML_CACHE/caniuse.json ]; then
   rm -f $HTML_CACHE/caniuse.json
   $QUIET || echo "Downloading caniuse data..."
   curl $($VERBOSE || echo "-s") \
-    -o $HTML_CACHE/caniuse.json -k \
```

Comment:
It works without it for me, but I've reverted this change, would need to look into why it was added and why it's no longer needed.